setup.py

```diff
@@ -43,7 +43,7 @@ def main():
             'Issue Tracker': 'https://github.com/googleapis/python-ndb/issues'
         },
         classifiers=[
-            "Development Status :: 4 - Beta",
+            "Development Status :: 5 - Production/Stable",
             "Intended Audience :: Developers",
             "License :: OSI Approved :: Apache Software License",
             "Programming Language :: Python",
```

Comment:
I just noticed this doesn't have a classifier for 3.8

Reply:
That's deliberate as our Kokoro setup doesn't run tests for 3.8. Let's handle that issue separately.